addons/hr_holidays/wizard/hr_leave_allocation_generate_multi_wizard.py

```diff
@@ -1,14 +1,15 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models, _
+from odoo import _, api, fields, models
 from odoo.addons.resource.models.utils import HOURS_PER_DAY
 
 
 class HrLeaveAllocationGenerateMultiWizard(models.TransientModel):
     _name = "hr.leave.allocation.generate.multi.wizard"
     _description = 'Generate time off allocations for multiple employees'
 
-    name = fields.Char("Description")
+    name = fields.Char("Description", compute="_compute_name", store=True, readonly=False)
+    is_name_custom = fields.Boolean(readonly=True, store=False)
```

Comment:
Remove that field, this is overkill. It's not a big deal if the name is overwritten.